homeassistant/components/tado/__init__.py

```diff
@@ -189,17 +189,12 @@ def decide_overlay_mode(
             return CONST_OVERLAY_TIMER
         # If no duration or timer set to fallback setting
         if overlay_mode is None:
-            overlay_mode = (
-                self.fallback if self.fallback is not None else CONST_OVERLAY_TADO_MODE
-            )
+            overlay_mode = self.fallback or CONST_OVERLAY_TADO_MODE
         # If default is Tado default then look it up
         if overlay_mode == CONST_OVERLAY_TADO_DEFAULT:
             zone_data = self.data["zone"][zone_id]
-            overlay_mode = (
-                zone_data.default_overlay_termination_type
-                if zone_data.default_overlay_termination_type is not None
-                else CONST_OVERLAY_TADO_MODE
-            )
+            zone_fallback = zone_data.default_overlay_termination_type
+            overlay_mode = zone_fallback or CONST_OVERLAY_TADO_MODE
```

Comment:
```suggestion
            overlay_mode = self.data["zone"][zone_id].default_overlay_termination_type or CONST_OVERLAY_TADO_MODE
```

Reply:
The `zone_data` and `zone_fallback` variables are not needed, since there's only one usage.

Reply:
Well, I created them to make the line smaller; otherwise, it runs into line length limits; hence, brackets are needed to have it multiline.(recommit addsw

Reply:
Brackets are fine, simplifying the code by removing variables would be preferred. Keep it short and simple. The line formatting is in line with the standards of Home Assistant.

Reply:
just committed the change

Reply:
Thanks @ethemcemozkan, looks good! :)